### DISCUSSION TITLE
Dichroic mirror supply

### DISCUSSION THREAD

**Discussion by wbrett87** [bionanoimaging/UC2-GIT#85] [Q&A]
Does anybody have experience with dichroic mirrors from [this](https://www.knightoptical.com/stock/default/filters/dichroic-filters/dichroic-longpass-filter.html) company? If I'm trying to image a fluorescent protein with excitation maximum @ 569nm and emission maximum at 594nm ([mScarlet](https://www.fpbase.org/protein/mscarlet/)), a 532nm laser and a 550nm longpass filter should do the trick, correct? How powerful does the laser need to be for widefield fluorescence?

**Comment by RainerHeintzmann** [comment]
No experience, but the price sounds interesting.
The trouble: the 500nm longpass still transmits about 10% 488nm for example.
If you have really good excitation and emission filters that may be OK.
They also do not seem to specify the surface flatness as in lambdas but this does not matter so much if you don't need to image the excitation.
A set by that company costs also more than 600€ and has only OD 5 overall suppression which is OK but not great.

**Comment by beniroquai** [comment]
Filters is indeed a tricky topic. We tested Edmund Optics, Comar and Thorlabs. Since Comar discontinues their off-the-shelf offers, we mostly rely on Thorlabs parts. One good point is that they have a large variety of dichroics and emission filters. With the laser - in most cases - you can safe the excitation filter (not necessarily for 635nm, they have 649nm sidebands.

**Reply by wbrett87** [reply]
Thanks! What kind of power output is acceptable for this scope? I ordered a 5mW laser to play around with, but my feeling is this is too weak based on the output on the laser you used in your version.

**Reply by beniroquai** [reply]
Depends on the fluorophore and the camera and of course the whole detection system. Start with some fluorescent marker pens to get some aha moments! For 532, the red one is the one of choice.

**Reply by wbrett87** [reply]
like [these](https://www.amazon.com/AGPtek-Fluorescent-Marker-Colors-Writing/dp/B008T1Z0HU)?

**Reply by wbrett87** [reply]
My understanding is both of these dichroics would work for my purpose, although the 550 may be optimal?
![image](https://user-images.githubusercontent.com/103341754/163207914-e780987b-18fc-4165-a697-c80b01e620ec.png)

**Reply by beniroquai** [reply]
The 550 should do the job. 
For the markers we use [those](https://www.amazon.com/Stabilo-BOSS-Original-Highlighter-Pastels/dp/B01LXOQ1KJ/ref=sr_1_3?crid=ZCZXEQW4LQW9&keywords=stabilo+highlighters&qid=1649914816&s=office-products&sprefix=stabilo+%2Coffice-products%2C118&sr=1-3)

**Comment by wbrett87** [comment]
Thanks for the input! Do you know of any sources of information on the design of fluorescence microscope systems that I could read? Preferably one that is accessible to a biologist.  I have a lot of questions, most of them probably pretty pedantic, so I don't want to overload this forum with them.

**Reply by beniroquai** [reply]
Hey, there is also a very nice youtube series available in the [iBiology channel](https://www.youtube.com/watch?v=AhzhOzgYoqw&ab_channel=iBiologyTechniques). Even one with Nico Sturman :) 

No worries about overloading anything. We wanted to have a "proper" forum for these kinda questions anyway!

**Reply by wbrett87** [reply]
The video looks like exactly what I need! Thanks for the link.

**Comment by RainerHeintzmann** [comment]
The latest edition of the handbook of biological confocal Microscopy, J. Pawley (Ed)
Is a recommended resource.

There is also a series of books by Cells (Ed.).

**Reply by wbrett87** [reply]
Many thanks!

**Comment by wbrett87** [comment]
Thank you kindly sir
